cfme/tests/infrastructure/test_provisioning_rest.py

```diff
@@ -57,14 +54,18 @@ def get_provision_data(rest_api, provider, template_name, auto_approve=True):
             "cc": "001"
         },
         "additional_values": {
-            "request_id": "1001"
+            "request_id": "1001",
+            "placement_auto": "true"
         },
         "ems_custom_attributes": {},
         "miq_custom_attributes": {}
     }
 
     if provider.one_of(RHEVMProvider):
         result["vm_fields"]["provision_type"] = "native_clone"
+        if current_version() > '5.9.0.16':
+            result["vm_fields"]["vlan"] = "<Template>"
```

Comment:
**QUESTION**: does this affect infra providers only?

Reply:
**Required**
Regardless, please use provider.appliance.version instead of current_version, even though its not a new import here.